checkov/common/runners/object_runner.py

```diff
@@ -138,3 +159,30 @@ def _change_files_path_to_relative(report: Report) -> None:
         for record in report.get_all_records():
             record.file_path = record.file_path.replace(os.getcwd(), "")
             record.resource = record.resource.replace(os.getcwd(), "")
+
+    def _get_triggers(self, definition: dict[str, Any]) -> set[str]:
+        triggers_set = set()
+        # it is correct that 'True' can be a key. It is easier to ingore the typing here,
+        # then to support it all the way up.
+        triggers = definition.get(True)  # type:ignore[call-overload]
+        try:
+            if triggers:
+                triggers_set = {key for key in triggers.keys() if key != START_LINE and key != END_LINE}
+
+        except Exception as e:
+            logging.info(f"Error:{str(e)}")
```

Comment:
@NoaAzoulay Please provide a more substantial text which explains where this error is being generated and what it means. i.e.: `failed to parse workflow triggers because of ...`, `failed to find triggers, file: ...` etc